src/app/components/publiclists/publiclists.component.scss

```diff
@@ -12,20 +20,13 @@
 }
 
 .web-container {
-  width: calc(100vw - 223px);
-  float: right;
+  background-color: #fafafa;
```

Comment:
Please revert all the changes of this file leaving this `background-color` change.